internal/host/validator.go

```diff
@@ -747,7 +751,11 @@ func (v *validator) printSufficientNetworkLatencyRequirementForRole(c *validatio
 			return fmt.Sprintf("Error while attempting to validate network latency: %s", err)
 		}
 		sort.Strings(hostNames)
-		return fmt.Sprintf("Network latency requirements of less or equals than %.3f ms not met for connectivity between %s and %s.", *c.clusterHostRequirements.Total.NetworkLatencyThresholdMs, c.host.RequestedHostname, strings.Join(hostNames, ","))
+		inv, err := hostutil.UnmarshalInventory(c.host.Inventory)
```

Comment:
The inventory is already part of the validationContext

Reply:
If it's part of the context, do you still think it's better to use hostID instead of the hostname in the error message? If I don't need to unmarshal the inventory, it is not going to be  expensive to retrieve the hostname.

Reply:
see my comment below, I will use the hostID instead.

Reply:
Done.